### PR TITLE
Update important-upgrade-notes.rst

### PR DESCRIPTION
**Pull request by amyblais**

### CODE REVIEW

**Review by wiggin77 — APPROVED** [review]
LGTM :+1:

**Review by streamer45 — APPROVED** [review]
Looks good :+1:

File: source/administration/important-upgrade-notes.rst [hunk 1]
```diff
@@ -16,6 +16,12 @@ Important Upgrade Notes
...
 +====================================================+==================================================================================================================================================================+
+| v5.33.0                                            | Deleting a reaction is now a soft delete in the ``Reactions`` table. A schema update is required and may take up to 15 seconds on first run with large data sets.|
+|                                                    +------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+|                                                    | Improved the websocket implementation by using epoll to manually read from a websocket connection. As a result, the number of goroutines is expected to go down  |
+|                                                    | by half. This implementation is only available on Linux and FreeBSD based distributions. If you are using nginx as a proxy to Mattermost, please ensure to have  |
```
Comment (cwarnermm) [inline source/administration/important-upgrade-notes.rst:22]:
```suggestion
|                                                    | by half. This implementation is only available on Linux and FreeBSD-based distributions. If you are using NGINX as a proxy to Mattermost, please ensure to have  |
```

**Review by cwarnermm — COMMENTED** [review]
Proposed updates provided inline.

**Pull request merged**